src/renderer/lib/hooks/useHandleDragStart.ts

```diff
@@ -0,0 +1,23 @@
+import { ipcRenderer } from "electron";
+import { useToasts } from "react-toast-notifications";
+
+export const useHandleDragStart = () => {
```

Comment:
The hook should represent what it's actually used for. In this case a more suitable name would be something like `useFileDrag`.

```suggestion
export const useFileDrag = () => {
```